compose/ui/ui-test-junit4/src/skikoMain/kotlin/androidx/compose/ui/test/ComposeUiTest.skikoMain.kt

```diff
@@ -229,6 +242,11 @@ class SkikoComposeUiTest(
         }
     }
 
+    // To simplify the tests porting from androidTest
+    fun setContentWithTestViewConfiguration(composable: @Composable () -> Unit) {
```

Comment:
Can you mark it as Internal API?

Reply:
It was used only in 3 tests :) So I simply removed this function for now and use `setContent`.